docs/knowledge_base/tips/multipage_application.md

```diff
@@ -0,0 +1,191 @@
+# Multi-page Application
+
+Are you tired of messy and confusing dashboards that make it hard to find the information you need? 
+If so, it might be a good idea to switch to multipage applications!
+
+![Multi-page Application](multipage_application.png){width=100%}
+
+Multipage applications allow you to organize your data and visualizations into separate pages, 
+making it easier to navigate and focus on specific insights. At Taipy, we understand 
+how important clear and effective data visualization is, 
+which is why we've created a multipage feature that lets you build user-friendly dashboards 
+that are easy to understand.
+
+We've prepared a two-part series of articles to explain how to use Taipy's multipage functionality:
+
+In Part 1, we'll go over the basics of creating a multipage application, including 
+how to set up your app and add pages.
+
+Stay tuned for Part 2, where we'll cover more advanced features like interactivity and customization. 
+Taipy makes creating multipage applications a breeze.
+
+So, if you're ready to enhance your data visualization, check out our multipage feature 
+and start creating your own intuitive and insightful applications with Taipy:
+
+## Part 1 – Building a Basic Multi-Page Application
+
+While it's possible to create a multi-page Taipy application in a single script, 
+it's often a good practice to organize your code into a folder structure like this:
+
+![Building a Basic Multi-Page Application](multipage_application_2.png){width=100%}
+
+In this arrangement, every submodule in the **pages** folder (like `home.py` and `temperature.py`) 
+holds the code for each page in our application. 
+We're demonstrating with just two pages in this example, but you can add as many as you need.
```

Comment:
I am not sure about that but what about adding a note about the default template that generates the multiple page structure automatically? @RymMichaut @FlorianJacta